fix mdtc

William1617 opened the pull request.

William1617 committed:
fix mdtc

mlxu995 commented:
Is your intention to remove redundant code？
but the value of "stack_num" not always bet set to 1, so we need to keep this variable to cover all the possibilities.

William1617 commented:
Ok.But the stack_num and stack_size in TCNStack are reversed

mlxu995 commented:
Thank you for your careful check. We'd really appreciate it if you help to fix this error.

mlxu995 committed:
refactor(model): 调整优化参数逻辑和顺序，提升可读性

- 调整 build_dilations 方法中循环顺序，先遍历stack_num再遍历stack_size
- 修改 stack_tcn_blocks 函数中 TCNStack 参数顺序，保持代码逻辑一致性的同时，提升代码可读性和维护性
